test/lit.cfg

```diff
@@ -384,6 +384,9 @@ config.substitutions.append(('%build-clang-importer-objc-overlays',
 config.substitutions.append(('%clang-importer-sdk-path',
                              '%r' % (make_path(config.test_source_root, 'Inputs', 'clang-importer-sdk'))))
 
+config.substitutions.append(('%clang-importer-sdk-frameworks',
```

Comment:
I think since this is always relative to `%clang-importer-sdk-path`, it's fine not to have a separate substitution for it. Substitutions are additional complexity if we're not actually making use of the flexibility.

Reply:
Aw, didn't know about the complexity of it. The reason added it was that it seems in the swift files, path to `clang-importer-sdk` is always hardcoded, like `%S/../../Inputs/clang-importer-sdk/...`, wanted to prevent that. But sounds good. Seems in the `lit.cfg` I can use `clang-importer-sdk-path`, but in the files, have to give it a relative path. I'll update the code with these changes.

Reply:
I would expect `%clang-importer-sdk-path/frameworks` to work in the individual files. What behavior are you seeing?

Reply:
That's what I also expected. It works fine with `lit.local.cfg` as `'-F %%clang-importer-sdk-path/frameworks '` but for using it in lit use cases like [here](https://github.com/maniramezan/swift/blob/master/test/ClangImporter/MixedSource/resolve-cross-language.swift#L3), `-F %clang-importer-sdk-path/frameworks` is turnt into `-F 'path-to-swift-repo/test/Inputs/clang-importer-sdk/' /frameworks`

Reply:
Hm. I would expect it to be `-F 'path-to-swift-repo/test/Inputs/clang-importer-sdk/'/frameworks` (no space), which should still work. Are you sure you didn't have a typo? We have [other examples](https://github.com/apple/swift/blob/master/test/Driver/emit-objc-header.swift#L3) of this kind of thing working.

Reply:
You're right, seems I was using different substitution that was causing that behavior. Thanks for the example, I went ahead and update the classes to use this pattern